docs/resources/Auto_Moderation.md

```diff
@@ -122,7 +122,11 @@ value of [action type](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-action-ob
 
 | Field      | Type       | Associated Action Types | Description                                    | 
 | ---------- | ---------- | ----------------------- | ---------------------------------------------- |
-| channel_id | snowflake  | SEND_ALERT_MESSAGE      | channel to which user content should be logged |
+| channel_id            | snowflake  | SEND_ALERT_MESSAGE      | channel to which user content should be logged              |
+| duration_seconds      | snowflake  | SEND_ALERT_MESSAGE      | timeout duration in seconds                                 |
+
+>info
+>Only trigger type 1 is permitted to use `duration_seconds` in metadata
```

Comment:
We should remove this? 🤔
It is clarified in `Associated Action Types` of action metadata. 

(I've replaced `SEND_ALERT_MESSAGE` to `TIMEOUT` locally)

Reply:
hm? it is not clarified, only keywords can use `TIMEOUT` maybe saying `keywords` is better than `action type 1`?

Reply:
Sorry! I misread it. All is right.